src/tally.F90

```diff
@@ -1555,13 +1626,51 @@ subroutine score_fission_eout_ce(p, t, i_score)
       ! change outgoing energy bin
       matching_bins(i) = binary_search(t % filters(i) % real_bins, n, E_out)
 
-      ! determine scoring index
-      i_filter = sum((matching_bins(1:t%n_filters) - 1) * t % stride) + 1
+      ! Case for tallying prompt neutrons
+      if (score_bin == SCORE_NU_FISSION .OR. \
```

Comment:
The backslash should be an ampersand. Also, continuation lines [should be indented by 5 spaces](http://openmc.readthedocs.io/en/latest/devguide/styleguide.html#indentation).